modules/bot.py

```diff
@@ -983,42 +1034,44 @@ def pick_player(self, member, args):
             return
 
         if len(args):
-            targetid = args[0].lstrip("<@!").rstrip(">")
-            if not targetid.isdigit():
+            target_id = args[0].lstrip("<@!").rstrip(">")
+            if not target_id.isdigit():
                 client.reply(self.channel, member, "You must specify a highlight!")
                 return
-            targetid = int(targetid)
-            for i in match.unpicked:
-                if i.id == targetid:
-                    team.append(i)
-                    match.unpicked.remove(i)
-                    if len(match.unpicked) == 0 or match.maxplayers == len(match.alpha_team) + len(match.beta_team):
-                        match.next_state()
-                    elif len(match.unpicked) == 1 and match.pick_order:
+            target_id = int(target_id)
+            if target_id <= 10:
+                player = match.unpicked_pool.pick_by_position(target_id)
+            else:
```

Comment:
This if-else is where we evaluate whether the `.pick <arg>` call provided a position as the arg or an @player as the arg. This part is untested whether it works correctly.

Reply:
:fyi: member_ids (which is what target_id is here) are usually long ints like 1234343919. Positions are likely only ever going to go up to 10 (1-10 + 2 captains = 12 players aka 6v6). We could bump this max possible position check to something like 20 to be safe.